### PR TITLE
ROX-12811: Grafana change datasource to prod-04 cluster and add a prod namespace

### PR DESCRIPTION
**Pull request by kurlov**
## Description

- Move from prod-01 to prod-04
- add prod namespace
- bump plugin

### CODE REVIEW

File: dashboards/grafana-dashboard-acs-fleet-manager-slos.configmap.yaml [hunk 1]
```diff
@@ -1008,9 +1008,14 @@ data:
                 "selected": true,
                 "text": "acs-fleet-manager-stage",
                 "value": "acs-fleet-manager-stage"
+              },
+              {
+                "selected": false,
```
Comment (ebensh) [inline dashboards/grafana-dashboard-acs-fleet-manager-slos.configmap.yaml:1013]: I'm unsure the larger context here, but I would expect the SLO dashboard config to default to prod over stage

File: dashboards/grafana-dashboard-acs-fleet-manager.configmap.yaml [hunk 2]
```diff
@@ -627,14 +626,15 @@ data:
             "name": "datasource",
             "options": [],
             "query": "prometheus",
+            "queryValue": "",
             "refresh": 1,
-            "regex": "app-sre-stage-01-prometheus|app-sre-prod-01-prometheus",
+            "regex": "app-sre-stage-01-prometheus|app-sre-prod-04-prometheus",
```
Comment (ebensh) [inline dashboards/grafana-dashboard-acs-fleet-manager.configmap.yaml:631]: Outside of this PR but are we supposed to be in stage-01 or stage-04 then?

File: dashboards/grafana-dashboard-acs-fleet-manager.configmap.yaml [hunk 3]
```diff
@@ -648,9 +648,14 @@ data:
                 "selected": true,
                 "text": "acs-fleet-manager-stage",
                 "value": "acs-fleet-manager-stage"
+              },
+              {
+                "selected": false,
```
Comment (ebensh) [inline dashboards/grafana-dashboard-acs-fleet-manager.configmap.yaml:653]: Same here - would expect default and current selected to be the prod instance

**Review by ebensh — CHANGES_REQUESTED** [review]

File: dashboards/grafana-dashboard-acs-fleet-manager.configmap.yaml (hunk 2 above)
Comment (kurlov) [reply dashboards/grafana-dashboard-acs-fleet-manager.configmap.yaml:631]: we have to be on stage-01 cluster

File: dashboards/grafana-dashboard-acs-fleet-manager-slos.configmap.yaml (hunk 1 above)
Comment (kurlov) [reply dashboards/grafana-dashboard-acs-fleet-manager-slos.configmap.yaml]: I don't have a strong preferences. Let's make prod a default one

**Review by ebensh — APPROVED** [review]

**Review by SimonBaeumer — APPROVED** [review]

**Pull request merged**